### ISSUE TITLE
Repo Code vs Manning Zip: Which one to use?

### ISSUE THREAD

**Issue by tajmone** [rust-in-action/code#36]
Hi Tim!

_Rust in Action_ recently went from MEAP to final release, but I've noticed that this repository doesn't always match the code examples mentioned in the book.

E.g., in __§2.1.1 Compiling single files with rustc__ it says:

>  You’ll find the source code for listing 2.1 in the file ch2/ok.rs

But the file `ch2/ok.rs` is only found in the code downloaded from Manning, not in this repo (see PR #34).

The Zip code from Manning is from MEAP 12 of the book. So I was wondering:

- Which should be the best code reference for the book, the Zip from Manning or this repository? 
- Will the repository be updated to mirror the final book? or is the code of the repository being developed independently?

**Comment by timClicks** [comment]
It's strange to hear that Manning hasn't updated the .zip file.

I will create the `ok.rs` file. I omitted it because I thought that it was too trivial. Sorry for the confusion!

**Comment by tajmone** [comment]
> It's strange to hear that Manning hasn't updated the .zip file.

Something didn't quite go right at Manning with the final book release, when I received the first email announcing _Rust in Action_ went out of MEAP, the ePub download was OK but the PDF would still produce the MEAP 16, and the code Zip is still MEAP 12. I wrote to them about it, and after a few weeks got a second email announcing for the book, and now the PDF was fixed.

> I will create the ok.rs file. I omitted it because I thought that it was too trivial. Sorry for the confusion!

I was really asking more out of concern about which code should be used, since the Zip from Manning is still marked as MEAP 12, but having seen a missing referenced file here I wasn't sure. Also, I noticed the sources in this repo also contain comments with AsciiDoc callouts in the sources, so I thought this repo was more your source folder for building the book.
